### PR TITLE
Request time

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/api/models.py [hunk 1]
```diff
@@ -593,6 +604,12 @@ def record_result(cls, event, result):
 
         api_user = get_api_user()
 
+        request_time = None
+        try:
+            request_time = int(result['request_time'])
```
Comment (nicpottier) [inline temba/api/models.py]: Why not `result.get('request_time')`? Then don't need try/except

File: templates/channels/channellog_log.haml [hunk 2]
```diff
@@ -3,6 +3,12 @@
     .time.pull-right
       {{log.created_on|date:"M j, Y g:i:s a"}}
 
+
+    .request-time.pull-right
+      Request time:
```
Comment (nicpottier) [inline templates/channels/channellog_log.haml]: Lets add this to the list page as well, and change errors to be red like we do on the channel log.

File: temba/api/models.py (hunk 1 above)
Comment (norkans7) [inline temba/api/models.py]: the try is for the int()

Comment (nicpottier) [inline temba/api/models.py]: How could that not succeed?

**Pull request merged**